lib/websocket-server.js

```diff
@@ -275,6 +275,14 @@ class WebSocketServer extends EventEmitter {
 
     cb(ws);
   }
+
+  /**
+   * Upgrade the connection to WebSocket.
+   * @public
+   */
+  port () {
```

Comment:
A couple of thoughts:

1. It's probably better to call this `address` and return `this._server.address()`. This will handle the case where the HTTP server is closed.
2. `WebSocketServer` could be initialized in `noServer` mode. In this case `this._server` is `null`. What should we return? Returning `null` doesn't make sense as it's possible that the external server is actually bound and listening for connections. Throwing an error is probably the best option.

Reply:
understood. Thank for the guide